examples/games/stepping.rs

```diff
@@ -52,7 +52,7 @@ impl Plugin for SteppingPlugin {
             ui_left: self.left,
             systems: Vec::new(),
         })
-        .add_systems(Startup, build_help)
+        .add_systems(Startup, build_stepping_hint)
```

Comment:
We should add this at the beginning of the `build()` function, and later return if feature is not enabled (like you did previously):

```rust
fn build(&self, app: &mut App) {
    app.add_systems(Startup, build_stepping_hint);
    if cfg!(not(feature="bevy_debug_stepping")) {
        return;
    }
        // ...
}
```

Otherwise we add unnecessary systems, and will get an error when pressing backquote without the "bevy_debug_stepping" feature